app/controllers/admin/users_controller.rb

```diff
@@ -3,7 +3,7 @@ class Admin::UsersController < Admin::BaseController
 
   def index
     name = params[:user_search] || ""
-    @users = User.filtered_by_name(name)
+    @users = User.filtered_by_name(name).user_not_admin
```

Comment:
```suggestion
    @users = User.normal_user.filtered_by_name(name)
```
enum có support cần j tạo scope